openapi.yaml

```diff
@@ -85,7 +85,7 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/ServiceInstanceAsyncProvision'
+                $ref: '#/components/schemas/ServiceInstanceAsyncOperation'
```

Comment:
Why are these changes part of this PR and not part of some other PR? Were they just missed or is there something in the release process that causes these changes to only happen now?